applications/GeoMechanicsApplication/tests/test_conditions.py

```diff
@@ -25,11 +25,11 @@ def check_water_pressure(self, test_name, etalon_value):
         pressure = test_helper.get_water_pressure(simulation)
         self.assertAlmostEqual(etalon_value, pressure[2])
 
-    def check_displacement(self, test_name, etalon_value):
+    def check_displacement(self, test_name, etalon_value, node_index=2):
```

Comment:
We could rename `node_index` to include that it's basically the node_index of the middle of the column, maybe something like (not sure, could be improved probably):
 ```suggestion
    def check_displacement(self, test_name, etalon_value, node_index_half_way_point=2):
```

Reply:
But it is not necessary to be at the half way. It can be everywhere, I just here as an optional point, selected the half way for these test cases. Don't you think to let it to be generic?

Reply:
It seems the displacement is always checked at the half way point in this test, so we could rename it, but I'm also fine with keeping it as is.